src/scss/overrides/_type.scss

```diff
@@ -1,191 +1,49 @@
-h1 {
-  margin: 20px 0;
-  font-size: 3rem;
-  line-height: 1;
-  letter-spacing: -0.1rem;
-  font-weight: 400;
-  color: $charcoal;
-  @include media-breakpoint-up(sm) {
-    line-height: 1.2;
-  }
-}
-
-.header-headline {
-  margin-bottom: 30px;
-  padding: 15px 0 10px 0;
-  border-bottom-width: 2px;
-  border-bottom-style: solid;
-}
-
-h2 {
-  margin: 20px 0 10px 0;
-  font-size: 2.8rem;
-  line-height: 1;
-  letter-spacing: -0.1rem;
-  font-weight: 200;
-  color: $charcoal;
-  @include media-breakpoint-up(sm) {
-    line-height: 1.2;
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
+  &:not(:first-child) {
+    margin-top: 1em;
```

Comment:
I'm thinking about this in the context of pulling unstyled content into our sites. For example, the wordpress API doesn't classes to hook on to, and I feel like having content columns (think `h1 + p + p + h2 + p + p` flow of elements) benefit from this significantly, while headings used for display purposes won't have a margin that then needs to be reset.

The Science site is a good example of this- heading elements at the top of pages and sections don't need margins, though their containers often have padding on them. Generally speaking, I see margin as a way to separate elements in a flow rather than for layout purposes due to the way the box model (`box-sizing: border-box`, of course) works.